clap_complete/tests/testsuite/engine.rs

```diff
@@ -682,14 +682,23 @@ fn suggest_multi_positional() {
     let mut cmd = Command::new("dynamic")
         .arg(
             clap::Arg::new("positional")
-                .value_parser(["pos_1, pos_2, pos_3"])
-                .index(1),
+                .value_parser(["pos_1"])
+                .index(1)
+                .required(true),
         )
         .arg(
             clap::Arg::new("positional-2")
                 .value_parser(["pos_a", "pos_b", "pos_c"])
                 .index(2)
-                .num_args(3),
+                .num_args(3)
+                .required(true),
```

Comment:
Why is `required` added?

Reply:
no idea

```
thread 'engine::suggest_multi_positional' panicked at clap_builder/src/builder/debug_asserts.rs:570:9:
Positional argument `[positional-3] [positional-3]...` *must* have `required(true)` or `last(true)` set because a prior positional argument (`<positional-2> <positional-2> <positional-2>`) has `num_args(1..)`
```